src/puppetlabs/puppetdb/cli/shovel.clj

```diff
@@ -0,0 +1,374 @@
+(ns puppetlabs.puppetdb.cli.shovel
```

Comment:
This namespace handles migrating from an "old" ActiveMQ version of PuppetDB to stockpile. My thinking here is taht we should be making a "best effort" to migrate existing messages. If we have some failure, or some messages can't be read (i.e. KahaDB corruption maybe) I don't think we should fail to start. Would be interested to hear if folks feel differently.